packages/shared/components/modals/AssetActionsModal.svelte

```diff
@@ -15,23 +15,23 @@
     import { MenuItem, Modal } from 'shared/components'
     import features from '@features/features'
 
-    export let modal: Modal = undefined
+    export let modal: Modal
     export let asset: IAsset
 
     function handleUnverify(): void {
         unverifyAsset(asset.id, NotVerifiedStatus.Skipped)
         updatePopupProps({
             asset: { ...asset, verification: { verified: false, status: NotVerifiedStatus.Skipped } },
         })
-        modal.close()
+        modal?.close()
```

Comment:
Is necessary the `?` . The model is no longer initialized to `undefined` by default, so it's required, right?
The same for the rest of `modal?.close()`

Reply:
my mistake, the type should be `Modal | undefined` as it takes the value from a dom element